storage/range.go

```diff
@@ -822,119 +858,64 @@ func (r *Range) applyRaftCommand(ctx context.Context, index uint64, originNode p
 		// since reads are served locally by the lease holder without going
 		// through Raft, a read which was not taken into account may have been
 		// served. Hence, we must retry at the current leader.
-		//
-		// It's crucial that we don't update the response cache for the error
```

Comment:
Why take out those lines about bypassing the response cache? Unless you know that by heart, you're likely going to be wondering for a minute whether or not it bypasses the response cache on purpose.